modules/ROOT/pages/api-sync.adoc

```diff
@@ -1,71 +1,58 @@
-= API Sync
+= Sync Your API Projects with API Sync
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: api, anypoint platform, sync, api sync
+:keywords: api, anypoint platform, sync, api sync, MuleSoft VCS
 
-The API Sync feature of Anypoint Studio enables you to:
+With API Sync, you can:
 
-* Start designing a RAML, or OAS 2.0 and 3.0 API specification in Studio and sync it with MuleSoft VCS.
-* Pull a published RAML or OAS 2.0 and 3.0 API specification from Exchange or a private Maven repository, into your Mule application project in Studio to scaffold a new project. +
-API sync also enables you to edit the API specification you imported from Exchange, and push your changes to MuleSoft VCS.
-+
-You can use API Sync on more than one API specification to scaffold multiple flows in different Mule configuration files.
-* Monitor the API versions that you implemented in your project and update your flows based on the latest published version.
-* Pull a RAML or OAS 2.0 and 3.0 API specification from MuleSoft VCS into Studio, edit the specification offline, and pull the updates back to MuleSoft VCS.
+* Start designing a RAML, or OAS 2.0 and 3.0 API spec in Studio and sync it with API Sync.
+* Pull a published RAML or OAS 2.0 and 3.0 API spec from Exchange or a private Maven repository, into your Mule application project in Studio to scaffold a new project. 
+* You can edit the API spec you imported from Exchange, and push your changes to MuleSoft VCS.
+* You can use API Sync on more than one API spec to scaffold multiple flows in different Mule configuration files.
+* Monitor the API versions that you implement in your project and update your flows based on the latest published version.
+* Pull a RAML or OAS 2.0 and 3.0 API spec from MuleSoft VCS into Studio, edit the spec offline, and pull the updates back to MuleSoft VCS.
+* Develop your Mule applications following API Lifecycle development practices.
 
-API Sync supports importing both RAML, OAS 2.0 and OAS 3.0 specifications.
+API Sync supports importing both RAML, OAS 2.0, and OAS 3.0 spec, comes bundled by default with Studio, and requires Mule 4.1.4 and later.
 
-API Sync comes bundled by default in Studio, and requires Mule 4.1.4 and later.
+API Sync can’t work properly if the RAML file exceeds 4,000 lines of text.
 
-API sync might not work properly if the RAML file exceeds 4,000 lines of text.
+== Create an API Spec in Studio
 
-Using API Sync, you can develop your Mule applications following API Lifecycle development practices.
+. Start a new API spec project in Studio.
++
+See xref:create-api-specification-studio.adoc[] for more information.
+. Publish it to Exchange.
++
+See xref:publish-api-project-to-exchange.adoc[]
+. Import the same API spec from Exchange to your Studio project.
```

Comment:
You should leave out teh last part of importing teh API Spec into Studio. It makes no sense as you already created the API in Studio and exported to Exchange

Reply:
The purpose of this doc is to explain how the API Sync feature works. 
If you think this is redundant, I can delete the last part.